tests/conftest.py

```diff
@@ -56,3 +56,24 @@ def create_list_model(list):
 
     new_lists = session.query(ListDB).all()
     return new_lists
+
+
+@pytest.fixture()
+def test_todo(session, test_list):
+    todo_data = [
+        {"title": "Test One", "details": "Test Details", "list_id": test_list[0].id, "completed": True},
+        {"title": "Test Two", "details": "Test Details", "list_id": test_list[1].id},
+        {"title": "Test No Description", "details": "", "list_id": test_list[2].id},
+    ]
+
+    def create_todo_model(todo):
+        return TodoDB(**todo)
+    
+    todos_map = map(create_todo_model, todo_data)
+    todos = list(todos_map)
+
+    session.add_all(todos)
+    session.commit()
+
+    new_todos = session.query(TodoDB).all()
+    return new_todos
```

Comment:
i think it makes sense to convert the returned todos into pydantic schemas before returning. It could be that what you have in the DB is different from what the user needs to send to your post endpoint

return [SomeSchema(**new_todo) for new_todo in new_todos]